personnel.md

```diff
@@ -0,0 +1,216 @@
+| [Drupal]                          | Larry Garfield ([@Crell])             |
+| [eZ Publish]                      | Andre Romcke ([@andrerom])            |
+| [Horde]                           | Jan Schneider ([@yunosh])             |
+| [IBM i Toolkit]                   | Adam Culp ([@adamculp])               |
+| [Icicle]                          | Aaron Piotrowski ([@trowski2002])     |
+| [Jackalope]                       | Lukas Kahwe Smith ([@lsmith])         |
+| [Joomla]                          | Robert Deutz ([@rdeutz])              |
+| [Lithium]                         | Nate Abele ([@nateabele])             |
+| [Magento]                         | Ben Marks ([@benmarks])               |
+| [PEAR]                            | Chuck Burgess ([@ashnazg])            |
+| [Phalcon]                         | Andres Gutierrez ([@phalconphp])      |
+| [Phing]                           | Michiel Rook ([@michieltcs])          |
+| [phpBB]                           | Marc Alexander ([@marc1706])          |
+| [phpDocumentor]                   | Mike van Riel ([@mvriel])             |
+| [PHPixie]                         | Roman Tsiupa ([@dracony_gimp])        |
+| [Pimcore]                         | Bernhard Rusch ([@bernhard_rusch])    |
+| [PPI Framework]                   | Paul Dragoonis ([@dr4goonis])         |
+| [PrestaShop]                      | Rémi Gaillard ([@xtaz07])             |
+| [PyroCMS]                         | Ryan Thompson ([@RyanThePyro])        |
+| [ReactPHP]                        | Cees-Jan Kiewiet ([@WyriHaximus])     |
+| [Revive Adserver]                 | Matteo Beccati ([@mbeccati])          |
+| [Sculpin]                         | Chris Tankersley ([@dragonmantank])   |
+| [Slim]                            | Jason Coward ([@drumshaman])          |
+| [SilverStripe]                    | Damian Mooyman                        |
+| [Stash]                           | Robert Hafner ([@tedivm])             |
+| [Stormpath PHP SDK]               | Brian Retterer ([@bretterer])         |
+| [SugarCRM]                        | Andreas Sandberg ([@yellowandy])      |
+| [Symfony]                         | Fabien Potencier ([@fabpot])          |
+| [Flow] and [Neos]                 | Karsten Dambekalns ([@kdambekalns])   |
+| [Wikibase] and [Semantic Media]   | Jeroen De Dauw ([@JeroenDeDauw])      |
+| [Yii framework]                   | Alexander Makarov ([@sam_dark])       |
+| [Zend Framework 2]                | Matthew Weier O'Phinney ([@mwop])     |
+| [Zikula]                          | Shefik Macauley ([@shefik_info])      |
+
+
+## Past
+
+### Former Secretaries
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| Amanda Folsom ([@ambassadorawsum])    | 2016-08-28 - 2017-11-12 |
+| Samantha Quiñones ([@ieatkillerbees]) | 2016-02-28 - 2016-12-24 |
+| Joe Ferguson ([@joepferguson])        | 2016-01-31 - 2016-06-29 |
+
```

Comment:
Also GeeH.

Reply:
Should I add him? I know he as been a secretary technically, but he wanted to withdraw his nomination and resigned immediately after the vote... Listing him with a 0 length term would strike as odd, I think.

Reply:
I think so yes.